src/mbus-master.cc

```diff
@@ -55,7 +56,11 @@ NAN_MODULE_INIT(MbusMaster::Init) {
 
     v8::Local<v8::Function> function = Nan::GetFunction(tpl).ToLocalChecked();
     constructor.Reset(function);
+#if NODE_MAJOR_VERSION >= 19
```

Comment:
is this really needed? (meaning the "if") This should be nan version wise, right? SO new nan have this also for older targets?

Reply:
I think the if is needed here since apparently in node versions above 18, `CreationContext()` has been changed to `GetCreationContext()`. I suppose that this is an API change in V8 to make its usage safer.

For demonstration purposes, I would temporarily revert the commit